frontend/javascripts/oxalis/controller/combinations/tool_controls.ts

```diff
@@ -619,6 +633,91 @@ export class BoundingBoxTool {
     getSceneController().highlightUserBoundingBox(null);
   }
 }
+
+export class QuickSelectTool {
+  static getPlaneMouseControls(
+    planeId: OrthoView,
+    planeView: PlaneView,
+    showNodeContextMenuAt: ShowContextMenuFunction,
+  ): any {
+    let startPos: Vector3 | null = null;
+    let currentPos: Vector3 | null = null;
+    let isDragging = false;
+    const SceneController = getSceneController();
+    const { rectangleGeometry } = SceneController;
+    return {
+      leftMouseDown: (pos: Point2, _plane: OrthoView, _event: MouseEvent) => {
+        const state = Store.getState();
+        rectangleGeometry.rotateToViewport();
+
+        const volumeTracing = getActiveSegmentationTracing(state);
+        if (!volumeTracing) {
+          return;
+        }
+
+        const [h, s, l] = getSegmentColorAsHSL(state, volumeTracing.activeCellId);
+        const activeCellColor = new THREE.Color().setHSL(h, s, l);
+        rectangleGeometry.setColor(activeCellColor);
+        startPos = V3.floor(calculateGlobalPos(state, pos));
+        currentPos = startPos;
+        isDragging = true;
+
+        Store.dispatch(confirmQuickSelectAction());
+        rectangleGeometry.unattachTexture();
```

Comment:
I added these comments to the attach/detach functions:

```
  attachTextureMask(ndData: Uint8Array, width: number, height: number) { 
    // Attach the array as a binary mask so that the rectangle preview
    // is only rendered where the passed array is 1.
    ...
  }

  detachTextureMask() {
    // Detach the texture mask, so that the full rectangle is visible again
    // (important while drawing the rectangle).
    ...
```